crates/turborepo-repository/src/change_mapper/mod.rs

```diff
@@ -29,6 +29,21 @@ pub enum LockfileChange {
     ChangedPackages(HashSet<turborepo_lockfiles::Package>),
 }
 
+/// This describes the state of a change to a lockfile.
+pub enum LockfileContents {
+    /// We know the lockfile did not change
+    Unchanged,
```

Comment:
although at first I thought this might not be needed because we have the list of files and can simply see that the lockfile wasn't in there, it's used in `change_detector` with `get_lockfile_contents`.